src/main/java/com/auction/retailService/entity/Store.java

```diff
@@ -0,0 +1,65 @@
+package com.auction.retailService.entity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@SequenceGenerator (name ="storeSeq", initialValue=1)
+@Table(name = "Store")
+public class Store {
+	
+	@ApiModelProperty(notes = "StoreID of the Store", hidden=true)
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator="storeSeq")
+	@Column (name = "store_id")
+	private Long storeID;
+	
+	@ApiModelProperty(notes = "Store name")
+	@NotNull
+	@Size (min = 3)
+	@Column (name = "store_Name")
```

Comment:
in this and several other cases you use a capitalize letter after the `_` this is not a valid snake_case format